src/vision/frc_vision_final.py

```diff
@@ -0,0 +1,190 @@
+# Contains most of the vision processing, though the most complete is under Offline_Filter
+# Likely to become the final product file for vision processing
+# The exposure options are likely needed to be adjusted per competition
+# They are currently tuned to the testing area
+# Tuning for the RealSense camera can be done easily through the ReaslSense Viewer app in a GUI
+# One thing I do still need to figure out is assigning the camera to a specific IO port number
+# Testing on my laptop, it is only visible when using one port
+
+from math import degrees, radians
+import pyrealsense2 as rs2
+import cv2
+import numpy as np
+import time
+from networktables import NetworkTables
+
+ANGLETHRESHOLD = 40
+
+# Takes in slopes x and y, tests if they are equal to each other or any previously verified line
+
+
+def unequal(new, old_list):
+    variance = 5
+    for i in old_list:
+        x3, y3, x4, y4 = i[0]
```

Comment:
Why are these `x3/4` instead of `x1/2`?

Reply:
It was just from when I was copying test code, before i made unequal into a function. that does probably make more sense though so I changed them around

Reply:
OK sure, it's not a big deal I was just confused.